### PR TITLE
Fix/tao 7652/rendering issues with tooltip in textreader

### PR DESCRIPTION
**Pull request by ricproenca-tao**
related to https://oat-sa.atlassian.net/browse/TAO-7652
need to backport to sprint95

### CODE REVIEW

File: views/scss/qti/_tooltip.scss [hunk 1]
```diff
@@ -25,4 +25,11 @@
 
 .qti-item .tooltip{
     padding: 10px;
+}
+
+.tooltip-active {
+    position: static;
+    border-style: none;
+    padding: 0 0 0 0;
```
Comment (jsconan) [inline views/scss/qti/_tooltip.scss]: You could just say: `padding: 0;`

File: views/scss/qti/_tooltip.scss [hunk 2]
```diff
@@ -25,4 +25,11 @@
...
+    font-size: 1em;
```
Comment (jsconan) [inline views/scss/qti/_tooltip.scss]: Is the `em` unit accurate here? Unless good reason (for instance size should be relative to parent), we should use `rem` instead.

File: views/scss/qti/_tooltip.scss [hunk 3]
```diff
@@ -25,4 +25,11 @@
 
 .qti-item .tooltip{
     padding: 10px;
+}
+
+.tooltip-active {
+    position: static;
```
Comment (jsconan) [inline views/scss/qti/_tooltip.scss]: Caution, this is introducing a side effect, the tooltip will be always placed at the same position in the page, and won't be displayed close to the related target.

image

File: views/scss/qti/_tooltip.scss (hunk 1 above)
Comment (ricproenca-tao) [reply views/scss/qti/_tooltip.scss]: yes

File: views/scss/qti/_tooltip.scss (hunk 2 above)
Comment (ricproenca-tao) [reply views/scss/qti/_tooltip.scss]: The `em` is accurate because in this case we want the to be size of the parent which should be the tetx size.

**Pull request merged**